src/test/kotlin/nl/info/zac/app/informatieobjecten/EnkelvoudigInformatieObjectRestServiceTest.kt

```diff
@@ -469,6 +474,42 @@ class EnkelvoudigInformatieObjectRestServiceTest : BehaviorSpec({
                 returnedEnkelvoudigInformatieObjectVersieGegevens shouldBe restEnkelvoudigInformatieObjectVersieGegevens
             }
         }
+        When("the enkelvoudig informatieobject is trying to be converted with status definitief") {
+            every {
+                enkelvoudigInformatieObjectConvertService.convertEnkelvoudigInformatieObject(
+                    any(), any()
+                )
+            } just Runs
+
+            enkelvoudiginformatieobject.status = StatusEnum.DEFINITIEF
+            val resp = enkelvoudigInformatieObjectRestService.convertInformatieObjectToPDF(
+                informatieobjectUUID,
+                zaak.uuid
+            )
+            Then("the response should be ok") {
+                resp.status shouldBe 200
+            }
+        }
+        When("the enkelvoudig informatieobject is trying to be converted with status in bewerking") {
+            every {
+                enkelvoudigInformatieObjectConvertService.convertEnkelvoudigInformatieObject(
+                    any(),
+                    any()
+                )
+            } throws ConvertException()
+
+            val resp = try {
+                enkelvoudigInformatieObjectRestService.convertInformatieObjectToPDF(informatieobjectUUID, zaak.uuid)
+            } catch (e: Exception) {
+                RestExceptionMapper().toResponse(e)
+            }
+
+            Then("the response should be an error message") {
+                resp.status shouldBe 400
+                val entity = resp.entity as String
+                entity shouldContain "\"message\":\"msg.error.convert.not.possible\""
```

Comment:
Literal strings are a bit easier to read usually.
```suggestion
                entity shouldContain """"message":"msg.error.convert.not.possible""""
```